### DISCUSSION TITLE
Settings - Express-p-11: Excluded Folders

### DISCUSSION THREAD

**Discussion by allisonfeeney** [usnistgov/easyexpress-public#1] [General]
In WG12 meeting 10/26/2023, Robert Swindels stated that there are other folders that could also be excluded from active workspace for STEP developers, and should be added to the default 
e.g.
ap236_furniture_catalog_and_interor_design 
modules unique to AP236. 

Are there others that the community would like to see excluded as a default?

**Comment by allisonfeeney** [comment]
AP236 Modules
- Parameterized_catalog_data_and_shape_representation
- Parameterized_catalog_data_information
- Furniture_interior_decoration
- Catalog_data_information_and_shape_representation
- Catalog_data_information
- Location_in_building
- Building_component
- Building_item
- Building_structure

**Comment by Tim-Suhr** [comment]
@allisonfeeney, 
I would think if ap203_configuration_controlled_3d_design_of_mechanical_parts_and_assemblies is not currently being excluded it should. As with any others that are now withdrawn.
